src/index.tsx

```diff
@@ -13,7 +13,8 @@ ReactDOM.render(
       integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
```

Comment:
@RobertPorterfield  -- Do we want to link to something hosted on SharePoint vs a Global CDN?

Reply:
Good question. I'm conflicted on it to be honest. On the one hand we have control over everything in SharePoint, and it guarantees if they can get to our page that it'll work. That's the track I've traditionally followed. On the other hand, caching... that integrity attribute (holds hash value of the linked file) gives us some measure of confidence that what's being pulled from the CDN is what we're looking for. Do you have strong feelings one way or the other?

Reply:
Issue #4